src/main/resources/assets/integrateddynamics/lang/en_US.lang

```diff
@@ -57,6 +57,7 @@ tile.blocks.integrateddynamics.dryingBasin.info=For drying items and fluids
 tile.blocks.integrateddynamics.squeezer.name=Squeezer
 tile.blocks.integrateddynamics.squeezer.info=Squeezes, flattens and crushes by jumping on it
 tile.blocks.integrateddynamics.menrilTorch.name=Menril Torch
+tile.blocks.integrateddynamics.invisibleLight.name=Invisible Light
```

Comment:
Since in the config screen this shows, and can be enabled/disabled, so i localized it